man/systemd-keyutil.xml

```diff
@@ -64,6 +64,16 @@
 
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><command>pkcs7</command></term>
+
+        <listitem><para>This commands converts an input <option>--signature=</option> in PKCS#1 format (RSA)
+        to an <option>--output=</option> file in PKCS#7 format (p7s) using the certificate given with
+        <option>--certificate=</option>.</para>
```

Comment:
"This commands generates a PKCS#7 signature that embeds the PKCS#1 signature provided with --signature=, ...."